docs/concepts/time-and-windows-in-ksql-queries.rst

```diff
@@ -180,36 +187,26 @@ windows per record key.
 
 When using windows in your KSQL queries, aggregate functions are applied only
 to the records that occur within a specific time window. Records that arrive
-late are handled as you might expect: although the time window they belong to
-has expired, the late records are still associated with the correct window.
+out-of-order are handled as you might expect: although the time window they belong to
+has expired, the out-of-order records are still associated with the correct window.

```

Comment:
I had to read this a few times to try to understand it and I'm still not sure I do.

What do you mean by 'has expired' here?  How, if at all, does expiry relate to the 'grace period' you mention below?

Do you mean `expired` as in the normal progression of timestamps in the stream of messages has moved beyond the end of the window?

Reply:
I agree with Andi, that "has expired" might not be the best terminology. What about `although the window end time passed,` ?